src/Nethermind/Nethermind.TxPool/Collections/BlobTxDistinctSortedPool.cs

```diff
@@ -1,20 +1,94 @@
 // SPDX-FileCopyrightText: 2023 Demerzel Solutions Limited
 // SPDX-License-Identifier: LGPL-3.0-only
 
+using System.Collections.Concurrent;
 using System.Collections.Generic;
 using Nethermind.Core;
 using Nethermind.Core.Crypto;
+using Nethermind.Core.Extensions;
+using Nethermind.Crypto;
 using Nethermind.Logging;
 
 namespace Nethermind.TxPool.Collections;
 
 public class BlobTxDistinctSortedPool(int capacity, IComparer<Transaction> comparer, ILogManager logManager)
     : TxDistinctSortedPool(capacity, comparer, logManager)
 {
+    protected override string ShortPoolName => "BlobPool";
+
+    internal ConcurrentDictionary<byte[], List<Hash256>> GetBlobIndex => _blobIndex;
```

Comment:
If it is property should be `BlobIndex`

Reply:
In current version `_blobIndex` is private and we use `GetBlobIndex` for external calls. So I think it should be `_blobIndex` then. But we can also drop `GetBlobIndex` and just use `_blobIndex` as property - and then rename to `BlobIndex`. I will change it